NethermindEth/helm-charts
# ci: add pre-commit and chart-testing

## Changes:

- Add `pre-commit` hooks config and workflow job
- Add chart-testing (`ct`) config and workflow job
- Add Makefile
- Update Contributing Guidelines with `pre-commit` instructions
- Add CODEOWNERS along with workflow check
